### DISCUSSION TITLE
Angular typescript Angular14

### DISCUSSION THREAD

**Discussion by anasmak04** [community/community#33582] [Other Feature Feedback, Questions, & Ideas] [answered]
Who is learning Angular in 2022?

**Comment by cosmicqbit** [comment]
A lot of people prefer Typescript with Angular. I might start with Typescript in a month or two.

**Reply by anasmak04** [reply]
Personally  I prefer to use. angular with typescript

**Comment by wesleygonalv** [comment, marked as answer]
I started to learn web development with Angular, and from all the other frameworks I've been through, Angular is the most structured and forces the developer to have layers and functionalities divisions

**Reply by anasmak04** [reply]
I agree with you it's more organized

**Comment by ng-druid** [comment]
What has worked well for our enterprise is rolling exclusively with Angular to realize our sustainable web content distribution [platform](https://github.com/rollthecloudinc/druid).
